NuGet.Config

```diff
@@ -3,5 +3,6 @@
   <packageSources>
     <clear />
     <add key="app-services-diagnostics-portal-nuget-feed" value="https://pkgs.dev.azure.com/app-service-diagnostics-portal/_packaging/app-services-diagnostics-portal-nuget-feed/nuget/v3/index.json" />
+    <add key="nuget" value="https://api.nuget.org/v3/index.json" />
```

Comment:
This might break the DevOps pipeline as I believe that doesn't allow multiple sources so revert this change